internal/service/advancedclustertpf/resource_schema.go

```diff
@@ -128,10 +127,7 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 				},
 			},
 			"create_date": schema.StringAttribute{
-				Computed: true,
-				PlanModifiers: []planmodifier.String{
-					stringplanmodifier.UseStateForUnknown(),
```

Comment:
it makes sense to remove `UseStateForUnknown`  as this is intended to use only when we're sure that value won't change